tests/integration/example/tests/sample1/sample1_test.go

```diff
@@ -45,9 +45,17 @@ func TestSample1(t *testing.T) {
 
 	var url string
 	if testEM.TestEnv.GetName() == appOnlyEnvName {
```

Comment:
shouldn't env and components exposed from TestEM via methods instead of direct access?